opensarlab-docs-jupyterbook/index.md

```diff
@@ -0,0 +1,57 @@
+# Welcome to OpenSARLab
+
+## What is OpenSARLab?
+
+OpenSARLab is a service providing users persistent, cloud-based, customizable computing environments. Groups of scientists and students have access to identical environments, containing the same software, running on the same hardware. It operates in the cloud, which means anyone with a moderately reliable internet connection can access their development environment. OpenSARLab sits alongside ASF's data archives in AWS, allowing for low latency transfer of large data products.
+
+OpenSARLab is a deployable service that creates an autoscaling Kubernetes cluster in Amazon AWS, running JupyterHub. Users have access to customizable environments running JupyterLab via authenticated accounts with persistent storage.
+
+While OpenSARLab was designed with SAR data science in mind, it is not limited to this field. Any group 
+development scenario involving large datasets and/or the need for complicated development environments
+can benefit from working in an OpenSARLab deployment.
+
+[Sign in or create an account to try OpenSARLab](https://opensarlab.asf.alaska.edu/hub/login)
```

Comment:
Update the link url to `https://opensciencelab.asf.alaska.edu`. Save the user one redirect trip.

Reply:
I'm not planning on making any content updates in this PR, pending merge of #49

Reply:
The new content updates in PR #49 move to Jupyter Book 2 and use MyST MD. I think this action builds with Jupyter Book 1, so not pulling those updates in on this PR probably makes sense. We may need to update the action when we want to build with Jupyter Book 2. When I tested it a couple of months ago, I had to generate a `myst.yml` from the `_toc.yml` and `_config.yml`.